rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/server/WebsocketServerTransport.java

```diff
@@ -107,33 +101,7 @@ public static WebsocketServerTransport create(InetSocketAddress address) {
   public static WebsocketServerTransport create(final HttpServer server) {
     Objects.requireNonNull(server, "server must not be null");
 
-    return new WebsocketServerTransport(
-        server.tcpConfiguration(
-            tcpServer ->
-                tcpServer.doOnConnection(
-                    connection ->
-                        connection.addHandlerLast(
-                            new ChannelInboundHandlerAdapter() {
-                              @Override
-                              public void channelRead(ChannelHandlerContext ctx, Object msg)
-                                  throws Exception {
-                                if (msg instanceof PongWebSocketFrame) {
-                                  logger.debug("received WebSocket Pong Frame");
-                                } else if (msg instanceof PingWebSocketFrame) {
```

Comment:
What is handling the incoming Ping frames after this PR?  Surely we still need to respond?

Reply:
pings are handled by reactor-netty https://github.com/reactor/reactor-netty/blob/cbd4c588abfb852931cb81d5a605f9ab5c844595/src/main/java/reactor/netty/http/server/WebsocketServerOperations.java#L147